hipcheck/src/main.rs

```diff
@@ -608,6 +612,46 @@ fn cmd_ready(config: &CliConfig) {
 	}
 }
 
+/// Run the Hipcheck self-updater to update to the latest release version.
+/// If the updater is not found, returns an error
+fn cmd_update(args: &UpdateArgs) {
+	// Try "hc-update" as the updater command, and see if it runs
+	let mut hc_command = Command::new("hc-update");
```

Comment:
It might be easier to use `which::which` here rather than trying the command and then trying the other name if it fails.

Reply:
Changed the code to do it that way.